internal/singleprogram/singleprogram.go

```diff
@@ -149,6 +149,14 @@ func Init(logger log.Logger) {
 		)
 	}
 
+	if _, err := exec.LookPath("src"); err != nil {
+		printStatusCheckError(
+			"src-cli is unavailable",
+			"Sourcegraph is better when src-cli is available; batch changes may not work.",
+			"Installation: https://github.com/sourcegraph/src-cli#sourcegraph-cli--",
```

Comment:
```suggestion
			"Installation: https://github.com/sourcegraph/src-cli",
```

Do we really need this hashtag given it is at the top of README 😂 (expscially with weird-but-correct `--` in the end